Engine/source/ts/collada/colladaShapeLoader.cpp

```diff
@@ -514,21 +514,21 @@ bool ColladaShapeLoader::canLoadCachedDTS(const Torque::Path& path)
    Torque::Path cachedPath(path);
    cachedPath.setExtension("cached.dts");
 
-   // Check if a cached DTS newer than this file is available
-   FileTime cachedModifyTime;
-   if (Platform::getFileTimes(cachedPath.getFullPath(), NULL, &cachedModifyTime))
-   {
-      bool forceLoadDAE = Con::getBoolVariable("$collada::forceLoadDAE", false);
```

Comment:
Hello! Your PR is good! Thanks for contribution.
You may want to restore this flag as it's used in the engine and as well from the editor to force the load of *.dae instead of cached *.dts.

Like:
if( Con::getBoolVariable("$collada::forceLoadDAE", false) )
   return false;

Then we can merge.
Thanks.